src/components/filters/Filters.jsx

```diff
@@ -0,0 +1,67 @@
+import React from 'react';
+import { Timeline, Icon, CustomSelect } from 'aqueduct-components';
+import { yearOptions, timeScaleOptions, spaceScaleOptions } from 'constants/filters';
+
+export default class Filters extends React.Component {
+  render() {
+    return (
+      <div className="c-filters">
+        {/* Year */}
+        <div className="c-select">
```

Comment:
I think that a grid is not needed to display a full-width element: .row > .col-sm-12 is the same as just drop the element and let it flow.